tests/datasets/entities/test_pluggable_entity.py

```diff
@@ -124,6 +124,8 @@ def build_request(query_body: Mapping[str, str]) -> Request:
         query=query,
         snql_anonymized=snql_anonymized,
         query_settings=HTTPQuerySettings(referrer=""),
-        attribution_info=AttributionInfo(get_app_id("blah"), "blah", None, None, None),
+        attribution_info=AttributionInfo(
+            get_app_id("blah"), {"blah": "blah"}, "blah", None, None, None
```

Comment:
let's have these fields be a little more descriptive of what they're supposed to be mimicking. e.g. `{"tenant_type": "tenant_id"}: